metricq_wizard_backend/metricq/network.py

```diff
@@ -0,0 +1,161 @@
+# metricq-wizard
+# Copyright (C) 2023 ZIH, Technische Universitaet Dresden, Federal Republic of Germany
+#
+# All rights reserved.
+#
+# This file is part of metricq-wizard.
+#
+# metricq-wizard is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# metricq-wizard is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with metricq-wizard.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
Place a warning in the module docstring about the following code that has been created during learning process.